README.md

```diff
@@ -15,30 +15,9 @@ All the official documentation on the Salesforce CLI Scanner plug-in is hosted o
 ### Document Link
 https://forcedotcom.github.io/sfdx-scanner/
 
+### Contributing
+Instructions [here](CONTRIBUTING.MD).
```

Comment:
relative linking is supposed to work according to this: https://github.blog/2013-01-31-relative-links-in-markup-files/
but I don't see the link working when I open the MD file from my branch. After merge, will test this again and fix it, if needed.

Reply:
I think this is because the extension is upper case instead of lowercase. Following the link here https://github.com/forcedotcom/sfdx-scanner/blob/d/rm/docBuildChange/README.md points to https://github.com/forcedotcom/sfdx-scanner/blob/d/rm/docBuildChange/CONTRIBUTING.MD which doesn't resolve.

Reply:
Good call. Let me update right here and try it out.

Reply:
That works, thanks!